app/(public)/revisions/compare/components/language-selector.tsx

```
@@ -0,0 +1,46 @@
+    <div className={`flex-shrink-0 ${className}`}>
+      <div className="relative">
+        <select
+          className="block w-full py-2 text-base border border-gray-300 focus:outline-none focus:ring-indigo-500 focus:border-indigo-500 sm:text-sm rounded-md"
+          value={currentLang}
+          onChange={handleLanguageChange}
+        >
+          {languages && languages.map((lang) => (
+            <option key={lang} value={lang}>
+              {lang.toUpperCase()}
+            </option>
+          ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)